From: =?utf-8?q?R=C3=A9mi_Palancher?= <remi-externe.palancher@edf.fr>
Date: Fri, 15 Dec 2017 09:16:03 +0100
Subject: Bump support to slurm 17.02.2

---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index f2c5dfd..69e39a7 100644
--- a/setup.py
+++ b/setup.py
@@ -24,7 +24,7 @@ logging.basicConfig(level=20)
 #VERSION = imp.load_source("/tmp", "pyslurm/__init__.py").__version__
 __version__ = "17.02.0"
 __min_slurm_hex_version__ = "0x110200"
-__max_slurm_hex_version__ = "0x110201"
+__max_slurm_hex_version__ = "0x110202"
 
 def fatal(logstring, code=1):
     logger.error("Fatal: " + logstring)
