From: Remi Palancher <remi-externe.palancher@edf.fr>
Date: Fri, 3 Jul 2015 15:10:30 +0200
Subject: [PATCH] fix-setup

Forwarded: No

---
 setup.py |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/setup.py b/setup.py
index c0347bd..a77462b 100644
--- a/setup.py
+++ b/setup.py
@@ -22,8 +22,8 @@ logging.basicConfig(level=20)
 # PySlurm Version
 
 #VERSION = imp.load_source("/tmp", "pyslurm/__init__.py").__version__
-__version__ = "14.11.5"
-__min_slurm_hex_version__ = "0x0e0b05"
+__version__ = "14.11.0-0"
+__min_slurm_hex_version__ = "0x0e0b00"
 __max_slurm_hex_version__ = "0x0e0b06"
 
 def fatal(logstring, code=1):
-- 
