From: =?utf-8?q?R=C3=A9mi_Palancher?= <remi@rezib.org>
Date: Mon, 29 Feb 2016 15:55:29 +0100
Subject: Enable build with 15.08.8

---
 setup.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/setup.py b/setup.py
index 931ed23..d539bf4 100644
--- a/setup.py
+++ b/setup.py
@@ -22,9 +22,9 @@ logging.basicConfig(level=20)
 # PySlurm Version
 
 #VERSION = imp.load_source("/tmp", "pyslurm/__init__.py").__version__
-__version__ = "15.08.2"
+__version__ = "15.08.8"
 __min_slurm_hex_version__ = "0x0f0802"
-__max_slurm_hex_version__ = "0x0f0803"
+__max_slurm_hex_version__ = "0x0f0808"
 
 def fatal(logstring, code=1):
 	logger.error("Fatal: " + logstring)
